src/components/progress-circle/progress-circle.spec.ts

```diff
@@ -42,18 +42,36 @@ describe('MdProgressCircular', () => {
       });
   });
 
-  it('should define a default value for the value attribute', (done: () => void) => {
+  it('should define a default value of undefined for the value attribute', (done: () => void) => {
     builder
       .overrideTemplate(TestApp, '<md-progress-circle></md-progress-circle>')
       .createAsync(TestApp)
       .then((fixture) => {
         fixture.detectChanges();
         let progressElement = getChildDebugElement(fixture.debugElement, 'md-progress-circle');
-        expect(progressElement.componentInstance.value).toBe(0);
+        expect(progressElement.componentInstance.value).toBe(undefined);
         done();
       });
   });
 
+  it('should set the value to undefined when the mode is set to indeterminate',
```

Comment:
Rather than using `done`, you can do the test like
```ts
it('should do a thing', async(() => {
  // do a bunch of stuff, some of which is async
}));
```

And it will magically know when its done.